src/styles.css

```diff
@@ -0,0 +1,142 @@
+/* You can add global styles to this file, and also import other style files */
```

Comment:
Are all of these styles reused in many different components? If there are styles specific to a component - they should be kept inside the style file of the component.